### PR TITLE
Set up routing

### PR DESCRIPTION
**Pull request by Elizabeth-Giovanniello**
Created pages for home page, feed, and account. Set up routing to each of those pages.

image
image
image

### CODE REVIEW

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+const HomePage = () => {
+    return (
```
Comment (catmcclelland) [inline src/pages/HomePage/HomePage.tsx:4]: very silly nit: either put a space between these two lines like the other pages or take the space out of the other pages

**Review by catmcclelland — APPROVED** [review]
LGTM except for 1 nit (see my comment regarding formatting)
Thank you so much for doing this 🥇

**Comment by jsmithb117** [comment]
Progress!!! I love it.

👍

File: src/App.tsx [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 function App() {
   // const [count, setCount] = useState(0)
 
   return (
-    <Button className="text-red-800 bg-blue-300 place-content-center" color="blue">Hello World</Button>
-    
+      <Routes>
+        <Route path="/feed" element={<FeedPage/>}/>
+        <Route path="/account" element={<AccountPage/>}/>
+        <Route path="/home" element={<HomePage/>}/>
```
Comment (jsmithb117) [inline src/App.tsx:15]: Are we going to have a landing page at ```path="/"``` or should we redirect ```"path="/"``` ```to="/home"```?

**Review by jsmithb117 — APPROVED** [review]
Progress!!! I love it.

👍

File: src/App.tsx (hunk 2 above)
Comment (Elizabeth-Giovanniello) [reply src/App.tsx:13]: In my mind "home" was going to be the landing page, so we can just change it to "/" instead of "/home"

**Pull request merged**